src/storage/mod.rs

```diff
@@ -1829,6 +1835,233 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_scan_with_key_only() {
+        let storage = TestStorageBuilder::new().build().unwrap();
```

Comment:
how about building it from titan engine

Reply:
Update PR to build from RockDB engine with Titan enbaled.